docs/src/main/tut/json.md

```diff
@@ -31,9 +31,9 @@ community.  The functionality is similar:
 
 ```scala
 libraryDependencies += Seq(
-  "org.http4s" %% "http4s-argonaut" % "{{< version >}}",
+  "org.http4s" %% "http4s-argonaut" % "{{< http4sVersion >}}",
   // Optional for auto-derivation of JSON codecs
-  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "1.2.0-M4"
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "1.2.0-M5"
```

Comment:
Good enough reasoning for me. Just wanted to make sure I wasn't skipping over a logical inconsistency.